### PR TITLE
changelog: add missing lib-help entry to 3.14.0

### PR DESCRIPTION
**Pull request by Blackbaud-ColbyWhite**

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Ahh, sorry for missing that when I released.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # 3.14.0 (2020-02-07)
 
 - Updated `skyux serve` to include the `host` properties from `skyuxconfig.json` in the Host URL.
+- Move help-widget logic into latest `@blackbaud/skyux-lib-help`. [#190](https://github.com/blackbaud/skyux-sdk-builder/pull/190)
```
Comment (Blackbaud-BobbyEarl) [inline CHANGELOG.md]: Can you make this past tense?

Comment (blackbaud-johnly) [reply CHANGELOG.md]: Good call. Also, should "help-widget" be a code snippet? If we aren't referring to code here, we don't need the hyphen. And we're also missing "the" before "latest."

**Comment by Blackbaud-BobbyEarl** [comment]
@blackbaud-johnly Nah, date should remain since it's already released.  This PR is fixing an oversight where I missed adding a change we made to the `CHANGELOG.md`.

**Pull request merged**